### PR TITLE
Dark steel conversion

### LINKED ISSUE #62: [Bug]: Loot chests spawn with Ender IO Dark Steel
### Issue Summary

Loot chests in the cities spawn with ender IO dark steel, but it's unusable since Gregtech dark steel is the usable one.  Either remove from loot tables, add a conversion recipe, or replace the loot in the chests.

### Reproduction steps

1.  Loot a city chest with dark steel
2. Obtain a useless ingot

### Relevant log output

_No response_

### PR DESCRIPTION
**Pull request by zcarp7220**
Fixes #62

### CODE REVIEW

File: kubejs/server_scripts/monicoins/saplings.js [hunk 1]
```diff
@@ -59,8 +59,8 @@ ServerEvents.recipes(event => {
     event.shaped(
         Item.of('minecraft:mangrove_propagule', 64), [
         'AA ',
-        ' A ',
-        '   '
+        '   ',
+        ' A '
```
Comment (LetterN) [inline kubejs/server_scripts/monicoins/saplings.js:63]: ??

File: kubejs/server_scripts/random_recipes.js [hunk 2]
```diff
@@ -601,4 +601,7 @@ ServerEvents.recipes(event => {
             });
         });
     });
+
+    //Dark Steel Conversion Recipie
+    event.shapeless('gtceu:dark_steel_ingot',['enderio:dark_steel_ingot'])
```
Comment (LetterN) [inline kubejs/server_scripts/random_recipes.js:606]: why? we have tags

**Review by LetterN — CHANGES_REQUESTED** [review]

**Comment by ThePansmith** [comment]
I do want to overhaul the loottables for the lost city chests anywho

File: kubejs/server_scripts/monicoins/saplings.js (hunk 1 above)
Comment (zcarp7220) [reply kubejs/server_scripts/monicoins/saplings.js:63]: I forgot to sync my repo when making this branch. It was an older commit thats aleardy here

**Pull request closed without merging**